packages/git-raw-commits/test.js

```diff
@@ -192,4 +190,24 @@ describe('git-raw-commits', function () {
         done()
       }))
   })
+
+  it('should emit an error if git is not available', function (done) {
+    try {
```

Comment:
Taking a second look at this, I think this `try`/`finally` _can_ be avoided with options to `rawExecOpts`... that would be much better.

Reply:
Nope, I was right the first time...

The reason for the `try`/`finally` is to ensure that changes to the process environment are always reverted, even in case the test fails to complete for some reason, so as to avoid affecting other tests.  My first thought was to pass environment options via `rawExecOpts` (see 58d1871221533815879e0f552e395413c8b027a1), but that is not possible without further code changes because [`cwd` is the only property of `rawExecOpts` that gets passed to `execFile`](https://github.com/conventional-changelog/conventional-changelog/blob/be1246c68f5dc4e6f28996129951a75bbf1cf307/packages/git-raw-commits/index.js#L48-L60).  This code is ugly because I erred on the side of making the test more complicated vs. changing more critical code.

I think it makes the most sense for `normalizeExecOpts` to set the `maxBuffer` property and use that return value (that includes all properties of the object passed as `rawExecOpts` after being normalized) to pass to `execFile`, but I'm not sure that is in scope for fixing this bug.